core/linux/src/internal/TimeZoneNative.kt

```diff
@@ -16,10 +18,43 @@ internal actual fun getAvailableZoneIds(): Set<String> =
 
 private val tzdb = runCatching { TzdbOnFilesystem() }
 
+@OptIn(ExperimentalForeignApi::class)
+private fun getTimezoneFromEtcTimezone(): String? {
```

Comment:
The logic in this function does not make it obvious why this is needed. Please add a short comment (like "this is needed in such and such versions of Debian") or a link to the issue that this function solves.